bin/provision-instances

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python3
+#
+# This script runs a full instance provision over all non-ephemeral instances
+# on OCIM. Running this script requires Django to be initialized - the easiest
+# way to do this is by evaluating the script from within the Django shell by
+# running:
```

Comment:
This is one possibility - another possibility would be to initialize Django via `django.setup()`. I wasn't sure if it would require more work to make sure Huey works normally, though.

Either way, since I was testing the bdero/rabbitmq branch separately on staging while writing this code, I didn't really have the time to fiddle with either of those options as I had to start provisioning the appservers overnight ASAP.

Maybe we can do this in a follow up ticket?